pkg/convert/json_ngxtranslate.go

```diff
@@ -0,0 +1,72 @@
+package convert
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+var ErrWrongKey = errors.New("Wrong key type")
+
+func fromNgxTranslate(b []byte) (messages model.Messages, err error) {
+	var dst map[string]interface{}
+
+	if err = json.Unmarshal(b, &dst); err != nil {
+		return messages, fmt.Errorf("unmarshal ngx translate: %w", err)
+	}
+
+	traverseMap := func(key string, value any) (err error) {
+		return err
+	}
+
+	traverseMap = func(key string, v any) (err error) {
+		switch value := v.(type) {
```

Comment:
Advisory. I'd swap names for consistency.

```suggestion
	traverseMap = func(key string, value interface{}) (err error) {
		switch v := value.(type) {
```